src/main/webapp/i18n/de/quizQuestionStatistic.json

```diff
@@ -2,21 +2,21 @@
     "artemisApp": {
         "quizQuestionStatistic": {
             "home": {
-                "title": "Question Statistics",
-                "createLabel": "Question Statistic erstellen"
+                "title": "Frage Statistken",
```

Comment:
```suggestion
                "title": "Fragestatistken",
```

Reply:
Sorry, my search and replace approach missed that occurrence, because it contains a typo :sweat_smile:. Fixed with commit 1e89fcd9c063d349e50675607d019336cdd6ac89.